tensorzero-core/src/endpoints/internal/config.rs

```diff
@@ -78,3 +80,45 @@ pub async fn get_config_by_hash_handler(
 
     Ok(Json(GetConfigResponse::from_snapshot(snapshot)))
 }
+
+/// Request body for writing a config snapshot.
+#[derive(Debug, Deserialize, Serialize)]
+pub struct WriteConfigRequest {
+    /// The config to write.
+    pub config: StoredConfig,
```

Comment:
why StoredConfig? this is the database type. Should we send `UninitializedConfig`?